build_tools/ci/run_matmul_test.sh

```diff
@@ -369,7 +375,8 @@ function run_matmul_test() {
   set +e
 
   compilation_flags="--iree-hal-target-backends=${target_backend} \
-                      --iree-amdaie-use-pipeline=${pipeline} \
+                      --iree-amdaie-use-lower-to-aie-pipeline=${lower_to_aie_pipeline} \
+                      --iree-amdaie-use-tile-pipeline=${tile_pipeline} \
```

Comment:
I think we don't have to keep `use` in the flag just to make the flag concise. I'm also okay if you want to keep it as is.

Reply:
Updated it